lib/spoom/sorbet/sigils.rb

```diff
@@ -0,0 +1,39 @@
+# typed: true
```

Comment:
What if we do `spoom bump lib/spoom/sorbet/sigils.rb`, does it bump itself to `strict`? :p

```suggestion
# typed: strict
```